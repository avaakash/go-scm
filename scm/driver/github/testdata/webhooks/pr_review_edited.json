{
    "action": "edited",
    "review": {
      "id": 3065773662,
      "node_id": "PRR_kwDOE_pLVc62u_5e",
      "user": {
        "login": "shivamnegi94",
        "id": 58415634,
        "node_id": "MDQ6VXNlcjU4NDE1NjM0",
        "avatar_url": "https://avatars.githubusercontent.com/u/58415634?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/shivamnegi94",
        "html_url": "https://github.com/shivamnegi94",
        "followers_url": "https://api.github.com/users/shivamnegi94/followers",
        "following_url": "https://api.github.com/users/shivamnegi94/following{/other_user}",
        "gists_url": "https://api.github.com/users/shivamnegi94/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/shivamnegi94/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/shivamnegi94/subscriptions",
        "organizations_url": "https://api.github.com/users/shivamnegi94/orgs",
        "repos_url": "https://api.github.com/users/shivamnegi94/repos",
        "events_url": "https://api.github.com/users/shivamnegi94/events{/privacy}",
        "received_events_url": "https://api.github.com/users/shivamnegi94/received_events",
        "type": "User",
        "user_view_type": "public",
        "site_admin": false
      },
      "body": "Gfhdddd",
      "commit_id": "9dc1b02c9b25bac779bdb2f31034f3f2421aa71d",
      "submitted_at": "2025-07-29T06:49:39Z",
      "state": "commented",
      "html_url": "https://github.com/shivamnegi94/testrepo/pull/206#pullrequestreview-3065773662",
      "pull_request_url": "https://api.github.com/repos/shivamnegi94/testrepo/pulls/206",
      "author_association": "OWNER",
      "_links": {
        "html": {
          "href": "https://github.com/shivamnegi94/testrepo/pull/206#pullrequestreview-3065773662"
        },
        "pull_request": {
          "href": "https://api.github.com/repos/shivamnegi94/testrepo/pulls/206"
        }
      }
    },
    "pull_request": {
      "url": "https://api.github.com/repos/shivamnegi94/testrepo/pulls/206",
      "id": 2702958367,
      "node_id": "PR_kwDOE_pLVc6hG98f",
      "html_url": "https://github.com/shivamnegi94/testrepo/pull/206",
      "diff_url": "https://github.com/shivamnegi94/testrepo/pull/206.diff",
      "patch_url": "https://github.com/shivamnegi94/testrepo/pull/206.patch",
      "issue_url": "https://api.github.com/repos/shivamnegi94/testrepo/issues/206",
      "number": 206,
      "state": "open",
      "locked": false,
      "title": "Update sacsac.yaml",
      "user": {
        "login": "utkarshchoubey-harness",
        "id": 101091340,
        "node_id": "U_kgDOBgaIDA",
        "avatar_url": "https://avatars.githubusercontent.com/u/101091340?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/utkarshchoubey-harness",
        "html_url": "https://github.com/utkarshchoubey-harness",
        "followers_url": "https://api.github.com/users/utkarshchoubey-harness/followers",
        "following_url": "https://api.github.com/users/utkarshchoubey-harness/following{/other_user}",
        "gists_url": "https://api.github.com/users/utkarshchoubey-harness/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/utkarshchoubey-harness/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/utkarshchoubey-harness/subscriptions",
        "organizations_url": "https://api.github.com/users/utkarshchoubey-harness/orgs",
        "repos_url": "https://api.github.com/users/utkarshchoubey-harness/repos",
        "events_url": "https://api.github.com/users/utkarshchoubey-harness/events{/privacy}",
        "received_events_url": "https://api.github.com/users/utkarshchoubey-harness/received_events",
        "type": "User",
        "user_view_type": "public",
        "site_admin": false
      },
      "body": null,
      "created_at": "2025-07-29T06:44:39Z",
      "updated_at": "2025-07-29T06:50:15Z",
      "closed_at": null,
      "merged_at": null,
      "merge_commit_sha": "10ff44855ecbf2933d09f92939d9fc6a205d335b",
      "assignee": null,
      "assignees": [
  
      ],
      "requested_reviewers": [
  
      ],
      "requested_teams": [
  
      ],
      "labels": [
  
      ],
      "milestone": null,
      "draft": false,
      "commits_url": "https://api.github.com/repos/shivamnegi94/testrepo/pulls/206/commits",
      "review_comments_url": "https://api.github.com/repos/shivamnegi94/testrepo/pulls/206/comments",
      "review_comment_url": "https://api.github.com/repos/shivamnegi94/testrepo/pulls/comments{/number}",
      "comments_url": "https://api.github.com/repos/shivamnegi94/testrepo/issues/206/comments",
      "statuses_url": "https://api.github.com/repos/shivamnegi94/testrepo/statuses/9dc1b02c9b25bac779bdb2f31034f3f2421aa71d",
      "head": {
        "label": "shivamnegi94:ut",
        "ref": "ut",
        "sha": "9dc1b02c9b25bac779bdb2f31034f3f2421aa71d",
        "user": {
          "login": "shivamnegi94",
          "id": 58415634,
          "node_id": "MDQ6VXNlcjU4NDE1NjM0",
          "avatar_url": "https://avatars.githubusercontent.com/u/58415634?v=4",
          "gravatar_id": "",
          "url": "https://api.github.com/users/shivamnegi94",
          "html_url": "https://github.com/shivamnegi94",
          "followers_url": "https://api.github.com/users/shivamnegi94/followers",
          "following_url": "https://api.github.com/users/shivamnegi94/following{/other_user}",
          "gists_url": "https://api.github.com/users/shivamnegi94/gists{/gist_id}",
          "starred_url": "https://api.github.com/users/shivamnegi94/starred{/owner}{/repo}",
          "subscriptions_url": "https://api.github.com/users/shivamnegi94/subscriptions",
          "organizations_url": "https://api.github.com/users/shivamnegi94/orgs",
          "repos_url": "https://api.github.com/users/shivamnegi94/repos",
          "events_url": "https://api.github.com/users/shivamnegi94/events{/privacy}",
          "received_events_url": "https://api.github.com/users/shivamnegi94/received_events",
          "type": "User",
          "user_view_type": "public",
          "site_admin": false
        },
        "repo": {
          "id": 335170389,
          "node_id": "MDEwOlJlcG9zaXRvcnkzMzUxNzAzODk=",
          "name": "testrepo",
          "full_name": "shivamnegi94/testrepo",
          "private": false,
          "owner": {
            "login": "shivamnegi94",
            "id": 58415634,
            "node_id": "MDQ6VXNlcjU4NDE1NjM0",
            "avatar_url": "https://avatars.githubusercontent.com/u/58415634?v=4",
            "gravatar_id": "",
            "url": "https://api.github.com/users/shivamnegi94",
            "html_url": "https://github.com/shivamnegi94",
            "followers_url": "https://api.github.com/users/shivamnegi94/followers",
            "following_url": "https://api.github.com/users/shivamnegi94/following{/other_user}",
            "gists_url": "https://api.github.com/users/shivamnegi94/gists{/gist_id}",
            "starred_url": "https://api.github.com/users/shivamnegi94/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/shivamnegi94/subscriptions",
            "organizations_url": "https://api.github.com/users/shivamnegi94/orgs",
            "repos_url": "https://api.github.com/users/shivamnegi94/repos",
            "events_url": "https://api.github.com/users/shivamnegi94/events{/privacy}",
            "received_events_url": "https://api.github.com/users/shivamnegi94/received_events",
            "type": "User",
            "user_view_type": "public",
            "site_admin": false
          },
          "html_url": "https://github.com/shivamnegi94/testrepo",
          "description": null,
          "fork": false,
          "url": "https://api.github.com/repos/shivamnegi94/testrepo",
          "forks_url": "https://api.github.com/repos/shivamnegi94/testrepo/forks",
          "keys_url": "https://api.github.com/repos/shivamnegi94/testrepo/keys{/key_id}",
          "collaborators_url": "https://api.github.com/repos/shivamnegi94/testrepo/collaborators{/collaborator}",
          "teams_url": "https://api.github.com/repos/shivamnegi94/testrepo/teams",
          "hooks_url": "https://api.github.com/repos/shivamnegi94/testrepo/hooks",
          "issue_events_url": "https://api.github.com/repos/shivamnegi94/testrepo/issues/events{/number}",
          "events_url": "https://api.github.com/repos/shivamnegi94/testrepo/events",
          "assignees_url": "https://api.github.com/repos/shivamnegi94/testrepo/assignees{/user}",
          "branches_url": "https://api.github.com/repos/shivamnegi94/testrepo/branches{/branch}",
          "tags_url": "https://api.github.com/repos/shivamnegi94/testrepo/tags",
          "blobs_url": "https://api.github.com/repos/shivamnegi94/testrepo/git/blobs{/sha}",
          "git_tags_url": "https://api.github.com/repos/shivamnegi94/testrepo/git/tags{/sha}",
          "git_refs_url": "https://api.github.com/repos/shivamnegi94/testrepo/git/refs{/sha}",
          "trees_url": "https://api.github.com/repos/shivamnegi94/testrepo/git/trees{/sha}",
          "statuses_url": "https://api.github.com/repos/shivamnegi94/testrepo/statuses/{sha}",
          "languages_url": "https://api.github.com/repos/shivamnegi94/testrepo/languages",
          "stargazers_url": "https://api.github.com/repos/shivamnegi94/testrepo/stargazers",
          "contributors_url": "https://api.github.com/repos/shivamnegi94/testrepo/contributors",
          "subscribers_url": "https://api.github.com/repos/shivamnegi94/testrepo/subscribers",
          "subscription_url": "https://api.github.com/repos/shivamnegi94/testrepo/subscription",
          "commits_url": "https://api.github.com/repos/shivamnegi94/testrepo/commits{/sha}",
          "git_commits_url": "https://api.github.com/repos/shivamnegi94/testrepo/git/commits{/sha}",
          "comments_url": "https://api.github.com/repos/shivamnegi94/testrepo/comments{/number}",
          "issue_comment_url": "https://api.github.com/repos/shivamnegi94/testrepo/issues/comments{/number}",
          "contents_url": "https://api.github.com/repos/shivamnegi94/testrepo/contents/{+path}",
          "compare_url": "https://api.github.com/repos/shivamnegi94/testrepo/compare/{base}...{head}",
          "merges_url": "https://api.github.com/repos/shivamnegi94/testrepo/merges",
          "archive_url": "https://api.github.com/repos/shivamnegi94/testrepo/{archive_format}{/ref}",
          "downloads_url": "https://api.github.com/repos/shivamnegi94/testrepo/downloads",
          "issues_url": "https://api.github.com/repos/shivamnegi94/testrepo/issues{/number}",
          "pulls_url": "https://api.github.com/repos/shivamnegi94/testrepo/pulls{/number}",
          "milestones_url": "https://api.github.com/repos/shivamnegi94/testrepo/milestones{/number}",
          "notifications_url": "https://api.github.com/repos/shivamnegi94/testrepo/notifications{?since,all,participating}",
          "labels_url": "https://api.github.com/repos/shivamnegi94/testrepo/labels{/name}",
          "releases_url": "https://api.github.com/repos/shivamnegi94/testrepo/releases{/id}",
          "deployments_url": "https://api.github.com/repos/shivamnegi94/testrepo/deployments",
          "created_at": "2021-02-02T04:41:22Z",
          "updated_at": "2025-07-29T06:43:46Z",
          "pushed_at": "2025-07-29T06:45:16Z",
          "git_url": "git://github.com/shivamnegi94/testrepo.git",
          "ssh_url": "git@github.com:shivamnegi94/testrepo.git",
          "clone_url": "https://github.com/shivamnegi94/testrepo.git",
          "svn_url": "https://github.com/shivamnegi94/testrepo",
          "homepage": null,
          "size": 333,
          "stargazers_count": 0,
          "watchers_count": 0,
          "language": null,
          "has_issues": true,
          "has_projects": true,
          "has_downloads": true,
          "has_wiki": true,
          "has_pages": false,
          "has_discussions": false,
          "forks_count": 0,
          "mirror_url": null,
          "archived": false,
          "disabled": false,
          "open_issues_count": 71,
          "license": null,
          "allow_forking": true,
          "is_template": false,
          "web_commit_signoff_required": false,
          "topics": [
  
          ],
          "visibility": "public",
          "forks": 0,
          "open_issues": 71,
          "watchers": 0,
          "default_branch": "main",
          "allow_squash_merge": true,
          "allow_merge_commit": true,
          "allow_rebase_merge": true,
          "allow_auto_merge": false,
          "delete_branch_on_merge": false,
          "allow_update_branch": false,
          "use_squash_pr_title_as_default": false,
          "squash_merge_commit_message": "COMMIT_MESSAGES",
          "squash_merge_commit_title": "COMMIT_OR_PR_TITLE",
          "merge_commit_message": "PR_TITLE",
          "merge_commit_title": "MERGE_MESSAGE"
        }
      },
      "base": {
        "label": "shivamnegi94:main",
        "ref": "main",
        "sha": "8115fbff29ce287c1c080ab132ccbb48dee76dcd",
        "user": {
          "login": "shivamnegi94",
          "id": 58415634,
          "node_id": "MDQ6VXNlcjU4NDE1NjM0",
          "avatar_url": "https://avatars.githubusercontent.com/u/58415634?v=4",
          "gravatar_id": "",
          "url": "https://api.github.com/users/shivamnegi94",
          "html_url": "https://github.com/shivamnegi94",
          "followers_url": "https://api.github.com/users/shivamnegi94/followers",
          "following_url": "https://api.github.com/users/shivamnegi94/following{/other_user}",
          "gists_url": "https://api.github.com/users/shivamnegi94/gists{/gist_id}",
          "starred_url": "https://api.github.com/users/shivamnegi94/starred{/owner}{/repo}",
          "subscriptions_url": "https://api.github.com/users/shivamnegi94/subscriptions",
          "organizations_url": "https://api.github.com/users/shivamnegi94/orgs",
          "repos_url": "https://api.github.com/users/shivamnegi94/repos",
          "events_url": "https://api.github.com/users/shivamnegi94/events{/privacy}",
          "received_events_url": "https://api.github.com/users/shivamnegi94/received_events",
          "type": "User",
          "user_view_type": "public",
          "site_admin": false
        },
        "repo": {
          "id": 335170389,
          "node_id": "MDEwOlJlcG9zaXRvcnkzMzUxNzAzODk=",
          "name": "testrepo",
          "full_name": "shivamnegi94/testrepo",
          "private": false,
          "owner": {
            "login": "shivamnegi94",
            "id": 58415634,
            "node_id": "MDQ6VXNlcjU4NDE1NjM0",
            "avatar_url": "https://avatars.githubusercontent.com/u/58415634?v=4",
            "gravatar_id": "",
            "url": "https://api.github.com/users/shivamnegi94",
            "html_url": "https://github.com/shivamnegi94",
            "followers_url": "https://api.github.com/users/shivamnegi94/followers",
            "following_url": "https://api.github.com/users/shivamnegi94/following{/other_user}",
            "gists_url": "https://api.github.com/users/shivamnegi94/gists{/gist_id}",
            "starred_url": "https://api.github.com/users/shivamnegi94/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/shivamnegi94/subscriptions",
            "organizations_url": "https://api.github.com/users/shivamnegi94/orgs",
            "repos_url": "https://api.github.com/users/shivamnegi94/repos",
            "events_url": "https://api.github.com/users/shivamnegi94/events{/privacy}",
            "received_events_url": "https://api.github.com/users/shivamnegi94/received_events",
            "type": "User",
            "user_view_type": "public",
            "site_admin": false
          },
          "html_url": "https://github.com/shivamnegi94/testrepo",
          "description": null,
          "fork": false,
          "url": "https://api.github.com/repos/shivamnegi94/testrepo",
          "forks_url": "https://api.github.com/repos/shivamnegi94/testrepo/forks",
          "keys_url": "https://api.github.com/repos/shivamnegi94/testrepo/keys{/key_id}",
          "collaborators_url": "https://api.github.com/repos/shivamnegi94/testrepo/collaborators{/collaborator}",
          "teams_url": "https://api.github.com/repos/shivamnegi94/testrepo/teams",
          "hooks_url": "https://api.github.com/repos/shivamnegi94/testrepo/hooks",
          "issue_events_url": "https://api.github.com/repos/shivamnegi94/testrepo/issues/events{/number}",
          "events_url": "https://api.github.com/repos/shivamnegi94/testrepo/events",
          "assignees_url": "https://api.github.com/repos/shivamnegi94/testrepo/assignees{/user}",
          "branches_url": "https://api.github.com/repos/shivamnegi94/testrepo/branches{/branch}",
          "tags_url": "https://api.github.com/repos/shivamnegi94/testrepo/tags",
          "blobs_url": "https://api.github.com/repos/shivamnegi94/testrepo/git/blobs{/sha}",
          "git_tags_url": "https://api.github.com/repos/shivamnegi94/testrepo/git/tags{/sha}",
          "git_refs_url": "https://api.github.com/repos/shivamnegi94/testrepo/git/refs{/sha}",
          "trees_url": "https://api.github.com/repos/shivamnegi94/testrepo/git/trees{/sha}",
          "statuses_url": "https://api.github.com/repos/shivamnegi94/testrepo/statuses/{sha}",
          "languages_url": "https://api.github.com/repos/shivamnegi94/testrepo/languages",
          "stargazers_url": "https://api.github.com/repos/shivamnegi94/testrepo/stargazers",
          "contributors_url": "https://api.github.com/repos/shivamnegi94/testrepo/contributors",
          "subscribers_url": "https://api.github.com/repos/shivamnegi94/testrepo/subscribers",
          "subscription_url": "https://api.github.com/repos/shivamnegi94/testrepo/subscription",
          "commits_url": "https://api.github.com/repos/shivamnegi94/testrepo/commits{/sha}",
          "git_commits_url": "https://api.github.com/repos/shivamnegi94/testrepo/git/commits{/sha}",
          "comments_url": "https://api.github.com/repos/shivamnegi94/testrepo/comments{/number}",
          "issue_comment_url": "https://api.github.com/repos/shivamnegi94/testrepo/issues/comments{/number}",
          "contents_url": "https://api.github.com/repos/shivamnegi94/testrepo/contents/{+path}",
          "compare_url": "https://api.github.com/repos/shivamnegi94/testrepo/compare/{base}...{head}",
          "merges_url": "https://api.github.com/repos/shivamnegi94/testrepo/merges",
          "archive_url": "https://api.github.com/repos/shivamnegi94/testrepo/{archive_format}{/ref}",
          "downloads_url": "https://api.github.com/repos/shivamnegi94/testrepo/downloads",
          "issues_url": "https://api.github.com/repos/shivamnegi94/testrepo/issues{/number}",
          "pulls_url": "https://api.github.com/repos/shivamnegi94/testrepo/pulls{/number}",
          "milestones_url": "https://api.github.com/repos/shivamnegi94/testrepo/milestones{/number}",
          "notifications_url": "https://api.github.com/repos/shivamnegi94/testrepo/notifications{?since,all,participating}",
          "labels_url": "https://api.github.com/repos/shivamnegi94/testrepo/labels{/name}",
          "releases_url": "https://api.github.com/repos/shivamnegi94/testrepo/releases{/id}",
          "deployments_url": "https://api.github.com/repos/shivamnegi94/testrepo/deployments",
          "created_at": "2021-02-02T04:41:22Z",
          "updated_at": "2025-07-29T06:43:46Z",
          "pushed_at": "2025-07-29T06:45:16Z",
          "git_url": "git://github.com/shivamnegi94/testrepo.git",
          "ssh_url": "git@github.com:shivamnegi94/testrepo.git",
          "clone_url": "https://github.com/shivamnegi94/testrepo.git",
          "svn_url": "https://github.com/shivamnegi94/testrepo",
          "homepage": null,
          "size": 333,
          "stargazers_count": 0,
          "watchers_count": 0,
          "language": null,
          "has_issues": true,
          "has_projects": true,
          "has_downloads": true,
          "has_wiki": true,
          "has_pages": false,
          "has_discussions": false,
          "forks_count": 0,
          "mirror_url": null,
          "archived": false,
          "disabled": false,
          "open_issues_count": 71,
          "license": null,
          "allow_forking": true,
          "is_template": false,
          "web_commit_signoff_required": false,
          "topics": [
  
          ],
          "visibility": "public",
          "forks": 0,
          "open_issues": 71,
          "watchers": 0,
          "default_branch": "main",
          "allow_squash_merge": true,
          "allow_merge_commit": true,
          "allow_rebase_merge": true,
          "allow_auto_merge": false,
          "delete_branch_on_merge": false,
          "allow_update_branch": false,
          "use_squash_pr_title_as_default": false,
          "squash_merge_commit_message": "COMMIT_MESSAGES",
          "squash_merge_commit_title": "COMMIT_OR_PR_TITLE",
          "merge_commit_message": "PR_TITLE",
          "merge_commit_title": "MERGE_MESSAGE"
        }
      },
      "_links": {
        "self": {
          "href": "https://api.github.com/repos/shivamnegi94/testrepo/pulls/206"
        },
        "html": {
          "href": "https://github.com/shivamnegi94/testrepo/pull/206"
        },
        "issue": {
          "href": "https://api.github.com/repos/shivamnegi94/testrepo/issues/206"
        },
        "comments": {
          "href": "https://api.github.com/repos/shivamnegi94/testrepo/issues/206/comments"
        },
        "review_comments": {
          "href": "https://api.github.com/repos/shivamnegi94/testrepo/pulls/206/comments"
        },
        "review_comment": {
          "href": "https://api.github.com/repos/shivamnegi94/testrepo/pulls/comments{/number}"
        },
        "commits": {
          "href": "https://api.github.com/repos/shivamnegi94/testrepo/pulls/206/commits"
        },
        "statuses": {
          "href": "https://api.github.com/repos/shivamnegi94/testrepo/statuses/9dc1b02c9b25bac779bdb2f31034f3f2421aa71d"
        }
      },
      "author_association": "COLLABORATOR",
      "auto_merge": null,
      "active_lock_reason": null
    },
    "changes": {
      "body": {
        "from": "Gfh"
      }
    },
    "repository": {
      "id": 335170389,
      "node_id": "MDEwOlJlcG9zaXRvcnkzMzUxNzAzODk=",
      "name": "testrepo",
      "full_name": "shivamnegi94/testrepo",
      "private": false,
      "owner": {
        "login": "shivamnegi94",
        "id": 58415634,
        "node_id": "MDQ6VXNlcjU4NDE1NjM0",
        "avatar_url": "https://avatars.githubusercontent.com/u/58415634?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/shivamnegi94",
        "html_url": "https://github.com/shivamnegi94",
        "followers_url": "https://api.github.com/users/shivamnegi94/followers",
        "following_url": "https://api.github.com/users/shivamnegi94/following{/other_user}",
        "gists_url": "https://api.github.com/users/shivamnegi94/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/shivamnegi94/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/shivamnegi94/subscriptions",
        "organizations_url": "https://api.github.com/users/shivamnegi94/orgs",
        "repos_url": "https://api.github.com/users/shivamnegi94/repos",
        "events_url": "https://api.github.com/users/shivamnegi94/events{/privacy}",
        "received_events_url": "https://api.github.com/users/shivamnegi94/received_events",
        "type": "User",
        "user_view_type": "public",
        "site_admin": false
      },
      "html_url": "https://github.com/shivamnegi94/testrepo",
      "description": null,
      "fork": false,
      "url": "https://api.github.com/repos/shivamnegi94/testrepo",
      "forks_url": "https://api.github.com/repos/shivamnegi94/testrepo/forks",
      "keys_url": "https://api.github.com/repos/shivamnegi94/testrepo/keys{/key_id}",
      "collaborators_url": "https://api.github.com/repos/shivamnegi94/testrepo/collaborators{/collaborator}",
      "teams_url": "https://api.github.com/repos/shivamnegi94/testrepo/teams",
      "hooks_url": "https://api.github.com/repos/shivamnegi94/testrepo/hooks",
      "issue_events_url": "https://api.github.com/repos/shivamnegi94/testrepo/issues/events{/number}",
      "events_url": "https://api.github.com/repos/shivamnegi94/testrepo/events",
      "assignees_url": "https://api.github.com/repos/shivamnegi94/testrepo/assignees{/user}",
      "branches_url": "https://api.github.com/repos/shivamnegi94/testrepo/branches{/branch}",
      "tags_url": "https://api.github.com/repos/shivamnegi94/testrepo/tags",
      "blobs_url": "https://api.github.com/repos/shivamnegi94/testrepo/git/blobs{/sha}",
      "git_tags_url": "https://api.github.com/repos/shivamnegi94/testrepo/git/tags{/sha}",
      "git_refs_url": "https://api.github.com/repos/shivamnegi94/testrepo/git/refs{/sha}",
      "trees_url": "https://api.github.com/repos/shivamnegi94/testrepo/git/trees{/sha}",
      "statuses_url": "https://api.github.com/repos/shivamnegi94/testrepo/statuses/{sha}",
      "languages_url": "https://api.github.com/repos/shivamnegi94/testrepo/languages",
      "stargazers_url": "https://api.github.com/repos/shivamnegi94/testrepo/stargazers",
      "contributors_url": "https://api.github.com/repos/shivamnegi94/testrepo/contributors",
      "subscribers_url": "https://api.github.com/repos/shivamnegi94/testrepo/subscribers",
      "subscription_url": "https://api.github.com/repos/shivamnegi94/testrepo/subscription",
      "commits_url": "https://api.github.com/repos/shivamnegi94/testrepo/commits{/sha}",
      "git_commits_url": "https://api.github.com/repos/shivamnegi94/testrepo/git/commits{/sha}",
      "comments_url": "https://api.github.com/repos/shivamnegi94/testrepo/comments{/number}",
      "issue_comment_url": "https://api.github.com/repos/shivamnegi94/testrepo/issues/comments{/number}",
      "contents_url": "https://api.github.com/repos/shivamnegi94/testrepo/contents/{+path}",
      "compare_url": "https://api.github.com/repos/shivamnegi94/testrepo/compare/{base}...{head}",
      "merges_url": "https://api.github.com/repos/shivamnegi94/testrepo/merges",
      "archive_url": "https://api.github.com/repos/shivamnegi94/testrepo/{archive_format}{/ref}",
      "downloads_url": "https://api.github.com/repos/shivamnegi94/testrepo/downloads",
      "issues_url": "https://api.github.com/repos/shivamnegi94/testrepo/issues{/number}",
      "pulls_url": "https://api.github.com/repos/shivamnegi94/testrepo/pulls{/number}",
      "milestones_url": "https://api.github.com/repos/shivamnegi94/testrepo/milestones{/number}",
      "notifications_url": "https://api.github.com/repos/shivamnegi94/testrepo/notifications{?since,all,participating}",
      "labels_url": "https://api.github.com/repos/shivamnegi94/testrepo/labels{/name}",
      "releases_url": "https://api.github.com/repos/shivamnegi94/testrepo/releases{/id}",
      "deployments_url": "https://api.github.com/repos/shivamnegi94/testrepo/deployments",
      "created_at": "2021-02-02T04:41:22Z",
      "updated_at": "2025-07-29T06:43:46Z",
      "pushed_at": "2025-07-29T06:45:16Z",
      "git_url": "git://github.com/shivamnegi94/testrepo.git",
      "ssh_url": "git@github.com:shivamnegi94/testrepo.git",
      "clone_url": "https://github.com/shivamnegi94/testrepo.git",
      "svn_url": "https://github.com/shivamnegi94/testrepo",
      "homepage": null,
      "size": 333,
      "stargazers_count": 0,
      "watchers_count": 0,
      "language": null,
      "has_issues": true,
      "has_projects": true,
      "has_downloads": true,
      "has_wiki": true,
      "has_pages": false,
      "has_discussions": false,
      "forks_count": 0,
      "mirror_url": null,
      "archived": false,
      "disabled": false,
      "open_issues_count": 71,
      "license": null,
      "allow_forking": true,
      "is_template": false,
      "web_commit_signoff_required": false,
      "topics": [
  
      ],
      "visibility": "public",
      "forks": 0,
      "open_issues": 71,
      "watchers": 0,
      "default_branch": "main"
    },
    "sender": {
      "login": "shivamnegi94",
      "id": 58415634,
      "node_id": "MDQ6VXNlcjU4NDE1NjM0",
      "avatar_url": "https://avatars.githubusercontent.com/u/58415634?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/shivamnegi94",
      "html_url": "https://github.com/shivamnegi94",
      "followers_url": "https://api.github.com/users/shivamnegi94/followers",
      "following_url": "https://api.github.com/users/shivamnegi94/following{/other_user}",
      "gists_url": "https://api.github.com/users/shivamnegi94/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/shivamnegi94/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/shivamnegi94/subscriptions",
      "organizations_url": "https://api.github.com/users/shivamnegi94/orgs",
      "repos_url": "https://api.github.com/users/shivamnegi94/repos",
      "events_url": "https://api.github.com/users/shivamnegi94/events{/privacy}",
      "received_events_url": "https://api.github.com/users/shivamnegi94/received_events",
      "type": "User",
      "user_view_type": "public",
      "site_admin": false
    }
  }