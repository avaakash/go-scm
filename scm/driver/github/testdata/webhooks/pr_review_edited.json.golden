{
  "Action": "edited",
  "Repo": {
    "ID": "335170389",
    "Namespace": "shivamnegi94",
    "Name": "testrepo",
    "Branch": "main",
    "Private": false,
    "Link": "https://github.com/shivamnegi94/testrepo",
    "Created": "2021-02-02T04:41:22Z",
    "Updated": "2025-07-29T06:43:46Z",
    "Visibility": 1,
    "Perm": {},
    "CloneSSH": "git@github.com:shivamnegi94/testrepo.git",
    "Clone": "https://github.com/shivamnegi94/testrepo.git"
  },
  "PullRequest": {
    "Number": 206,
    "Title": "Update sacsac.yaml",
    "Body": null,
    "Sha": "9dc1b02c9b25bac779bdb2f31034f3f2421aa71d",
    "Ref": "refs/pull/206/head",
    "Source": "ut",
    "Target": "main",
    "Fork": "shivamnegi94/testrepo",
    "Link": "https://github.com/shivamnegi94/testrepo/pull/206",
    "Diff": "https://github.com/shivamnegi94/testrepo/pull/206.diff",
    "Closed": false,
    "Merged": false,
    "Base": {
      "Sha": "8115fbff29ce287c1c080ab132ccbb48dee76dcd",
      "Path": "refs/heads/main",
      "Name": "main"
    },
    "Head": {
      "Sha": "9dc1b02c9b25bac779bdb2f31034f3f2421aa71d",
      "Path": "refs/heads/ut",
      "Name": "ut"
    },
    "Author": {
      "Login": "utkarshchoubey-harness",
      "Name": "",
      "Email": "",
      "Avatar": "https://avatars.githubusercontent.com/u/101091340?v=4"
    },
    "Created": "2025-07-29T06:44:39Z",
    "Updated": "2025-07-29T06:50:15Z"
  },
  "Sender": {
    "Login": "shivamnegi94",
    "Name": "",
    "Email": "",
    "Avatar": "https://avatars.githubusercontent.com/u/58415634?v=4"
  }
}
